### ISSUE TITLE
Any working example for offline usage?

### ISSUE THREAD

**Issue by radiegtya** [feathersjs/feathers-redux#26]
Sorry for asking such question, but I found difficulty about implementing the feathers-redux-offline. Some simple working example would be appreciated.

Thanks

**Comment by eddyystop** [comment]
Are you are asking about mutations while the client is not connected to the server? That is not yet supported.

Mobile support must first be implemented in feathers-offline-log. It would be great if you could contribute to that.

**Comment by radiegtya** [comment]
Nope, I just need to fetch data (Find or get) while offline.. I don't need to mutate anything..
And I need the example, can you provide it @eddyystop ?

**Comment by eddyystop** [comment]
That is covered in the docs https://docs.feathersjs.com/guides/offline-first/example-optimistic.html
